tools/crm_resource.c

```diff
@@ -472,6 +472,7 @@ main(int argc, char **argv)
                 break;
 
             case 'C':
+                require_resource = FALSE; 
             case 'R':
             case 'P':
                 crm_log_args(argc, argv);
```

Comment:
It doesn't hurt to set require_resource = FALSE under 'C', but it's not necessary, because execution will fall through to where it is already set just below this.

FYI, 'R' and 'P' are a special case. Just below this, you can see where rsc_cmd is set to 'C', regardless of what option the user specified. So, 'R' and 'P' have the identical requirements as 'C'.